projects/packages/classic-theme-helper/src/class-social-links.php

```diff
@@ -82,41 +87,41 @@ public function __construct() {
 				return;
 			}
 
-			$this->theme_supported_services = $theme_support[0];
-			$this->links                    = class_exists( \Jetpack_Options::class ) ? \Jetpack_Options::get_option( 'social_links', array() ) : '';
+			self::$theme_supported_services = $theme_support[0];
+			self::$links                    = class_exists( Jetpack_Options::class ) ? Jetpack_Options::get_option( 'social_links', array() ) : '';
 
-			$this->admin_setup();
+			self::admin_setup();
 
-			add_filter( 'jetpack_has_social_links', array( $this, 'has_social_links' ) );
-			add_filter( 'jetpack_get_social_links', array( $this, 'get_social_links' ) );
+			add_filter( 'jetpack_has_social_links', array( __CLASS__, 'has_social_links' ) );
+			add_filter( 'jetpack_get_social_links', array( __CLASS__, 'get_social_links' ) );
 
 			foreach ( $theme_support[0] as $service ) {
-				add_filter( "pre_option_jetpack-$service", array( $this, 'get_social_link_filter' ) ); // - `get_option( 'jetpack-service' );`
-				add_filter( "theme_mod_jetpack-$service", array( $this, 'get_social_link_filter' ) ); // - `get_theme_mod( 'jetpack-service' );`
+				add_filter( "pre_option_jetpack-$service", array( __CLASS__, 'get_social_link_filter' ) ); // - `get_option( 'jetpack-service' );`
+				add_filter( "theme_mod_jetpack-$service", array( __CLASS__, 'get_social_link_filter' ) ); // - `get_theme_mod( 'jetpack-service' );`
 			}
 		}
 
 		/**
 		 * Init the admin setup.
 		 */
-		public function admin_setup() {
+		public static function admin_setup() {
 			if ( ! current_user_can( 'manage_options' ) ) {
 				return;
 			}
 
-			if ( ! is_admin() && ! $this->is_customize_preview() ) {
+			if ( ! is_admin() && ! self::is_customize_preview() ) {
 				return;
 			}
 
-			// @phan-suppress-next-line PhanUndeclaredFunction -- Function checked with function_exists - see https://github.com/phan/phan/issues/1204.
-			$this->publicize    = function_exists( 'publicize_init' ) ? publicize_init() : null;
-			$publicize_services = $this->publicize->get_services( 'connected' );
-			$this->services     = array_intersect( array_keys( $publicize_services ), $this->theme_supported_services );
+			// @phan-suppress-next-line PhanUndeclaredFunction -- See init() for the condition that ensures this function exists.
+			self::$publicize    = publicize_init();
```

Comment:
The change to this line is because `publicize_init` (here in `admin_setup`) can't be null, because if it was we would have returned in the first if block within the `init` function.